python/ray/serve/controller.py

```diff
@@ -358,6 +358,22 @@ def deploy(
 
         return goal_id, updating
 
+    def deploy_group(
+        self, deployment_args_list: List[Dict]
+    ) -> List[Tuple[Optional[GoalId], bool]]:
+        """
+        Takes in a list of dictionaries that contain keyword arguments for the
+        controller's deploy() function. Calls deploy on all the argument
```

Comment:
Agreed. We have an [issue filed](https://github.com/ray-project/ray/issues/21879) to make a configuration schema object for deployments with Pydantic validation. Once that's resolved, we can refactor these functions to use that object instead of a dictionary of parameters.